ext/win32ole/win32ole.c

```diff
@@ -687,10 +687,19 @@ ole_cp2encoding(UINT cp)
         }
     }
 
-    enc_name = rb_sprintf("CP%d", cp);
-    idx = rb_enc_find_index(enc_cstr = StringValueCStr(enc_name));
+    switch(cp) {
+      case 20932:
+        enc_cstr = "eucJP-ms";
```

Comment:
Maybe we should handle other mappings from `ole_encoding2cp()`? See https://github.com/ruby/ruby/blob/2aa8edaec701ca2d76e7be7a9b792b69574ac869/ext/win32ole/win32ole.c#L491